aosp_diff/preliminary/external/drm_hwcomposer/0006-fix-static-bound-to-dynamically-bound.patch

```diff
@@ -0,0 +1,216 @@
+From a3ba2d68322751d30502a12e4f28d8699f38a9fd Mon Sep 17 00:00:00 2001
+From: Xiao HaowenX <haowenx.xiao@intel.com>
+Date: Fri, 12 Nov 2021 10:51:51 +0800
+Subject: [PATCH] fix static bound to dynamically bound
+
+Tracked-On:OAM-99492
+Signed-off-by: Xiao HaowenX <haowenx.xiao@intel.com>
+---
+ DrmHwcTwo.cpp         | 37 ++++++++++++++++++++++++++++++++-----
+ DrmHwcTwo.h           |  1 +
+ drm/DrmDevice.cpp     | 41 +++++------------------------------------
+ drm/DrmDevice.h       |  7 ++++---
+ drm/ResourceManager.h |  3 +++
+ 5 files changed, 45 insertions(+), 44 deletions(-)
+
+diff --git a/DrmHwcTwo.cpp b/DrmHwcTwo.cpp
+index 194b11e..63ee007 100644
+--- a/DrmHwcTwo.cpp
++++ b/DrmHwcTwo.cpp
+@@ -1202,23 +1202,47 @@ void DrmHwcTwo::HandleInitialHotplugState(DrmDevice *drmDevice) {
+   }
+ }
+ 
++int DrmHwcTwo::DynamicallyBound(DrmConnector *mconn,DrmDevice *mdrm){
++	int curr_num_display = resource_manager_.getDisplayCount();
++	mconn->set_display(curr_num_display);
```

Comment:
Need check max_pipe_count. Connectors counter is bigger than crtc. We need detect the case that hotplug more monitors than crtc counter.